StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/LinkUI.swift

```diff
@@ -199,6 +199,7 @@ extension LinkUI {
         // Text
         appearance.colors.text = .linkTextPrimary
         appearance.colors.textSecondary = .linkTextSecondary
+        appearance.colors.brandText = .linkTextBrand
```

Comment:
Calling this out as this is the first usage of a newly added `brandText` in this PR. This was added because the "Resend Code" button on the link verification view was hardcoded to use a color not present in `PaymentSheet.Appearance`.